picard/tagger.py

```diff
@@ -360,9 +445,106 @@ def load_to_picard(self, items):
             for item in parsed_items.mbids | parsed_items.urls:
                 thread.to_main(file_lookup.mbid_lookup, item, None, None, False)
 
-        if parsed_items:
+        for command in parsed_items.commands:
+            self.handle_command(command)
+
+        if parsed_items.non_executable_items():
             self.bring_tagger_front()
 
+    def get_album_pane_tracks(self):
+        for album in self.albums.values():
+            for track in album.iterfiles():
+                yield track
```

Comment:
This could be

```python
yield from album.iterfiles()
```

Or even the entire method could also be done with:

```python
def get_album_pane_tracks(self):
   return iter_files_from_objects(self.albums)
```

Also I find the name a bit misleading, it is not yielding Track objects, but files, so maybe also name this ` iter_album_files`?